samples/msgext-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,102 @@
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Select **File > Open Folder** in VS Code and choose this samples directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+## Setup and use the sample
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+1) Navigate to the `samples/msgext-doc-compliance-checker` folder and open with Visual Studio Code.
+
+1) Navigate to the `samples/msgext-doc-compliance-checker/.localConfigs` directory and update the values below.
+
+   ```txt
+      END_POINT="https://<your-service>.azurewebsites.net"
+      API_KEY="your-api-key"
+      DEPLOYMENT_ID="your-deployment-id"
+      AZURE_STORAGE_CONNECTION_STRING="The connection string for your Azure Storage account."
+      CONTAINER_NAME="your-container"
+      CHECKLIST_NAME="your-checklist-name"
+   ```
+
+## Running the sample
+
+**Select Upload an app**
+![UploadCustomeApp](Images/0.UploadCustomeApp.png)
+
+**Select Add**
+![ComplianceCheckerInstallApp](Images/1.ComplianceCheckerInstallApp.png)
+
+**Search for your app in Messaging Extensions**
+![ComplianceCheckerSearchApp](Images/2.ComplianceCheckerSearchApp.png)
+
+**Select App**
+![ComplianceCheckerSelectApp](Images/3.ComplianceCheckerSelectApp.png)
+
+**Search File Name**
+![ComplianceCheckerSearchFileName](Images/4.ComplianceCheckerSearchFileName.png)
+
+**File Name Results**
+![ComplianceCheckerSelectResults](Images/5.ComplianceCheckerSelectResults.png)
+
+**Compliance Checker Results**
+![6.ComplianceCheckerResults](Images/6.ComplianceCheckerResults.png)
+
+**Compliance Checker Yes Results**
+![7.ComplianceCheckerYesResults](Images/7.ComplianceCheckerYesResults.png)
+
+**Compliance Checker No Results**
+![8.ComplianceCheckerNoResults](Images/8.ComplianceCheckerNoResults.png)
+
+### Test the app in Copilot for Microsoft 365
+Navigate to the Microsoft Copilot for Microsoft 365 chat. Check the lower left of the chat user interface, below the compose box. You should see a plugin icon. Click this and enable the msgext-doc-compliance-checker plugin.
```

Comment:
I cant see the prompts to test the sample in Copilot.

2-3 Realtime prompts should also be documented.

Reply:
Copilot still not tested.

Reply:
Tested Its Working